basic-data-structures/check-if-an-object-has-a-property.js

```diff
@@ -0,0 +1,35 @@
+let users = {
+  Alan: {
+    age: 27,
+    online: true
+  },
+  Jeff: {
+    age: 32,
+    online: true
+  },
+  Sarah: {
+    age: 48,
+    online: true
+  },
+  Ryan: {
+    age: 19,
+    online: true
+  }
+};
+
+function isEveryoneHere(userObj) {
+  // Only change code below this line
+  let namesOfUsers = ["Alan", "Jeff", "Sarah", "Ryan"];
+
+  for (let i = 0; i < namesOfUsers.length; i++) {
+    let names = namesOfUsers[i];
+    if (userObj.hasOwnProperty(names) === false) {
```

Comment:
This structure is not good.
As userObj.hasOwnProperty(names) is already a boolean you dont need to test if it's equal false and return false
you can use the negation character (!) to build a cleaner logic
for example: (!usersObj.hasOwnProperty(names)) -> means that false wil became true and inside the if structure return false. 

Other thing, I think name in userObj a cleaner way to write that instead of use the method hasOwnProperty so change this too.